examples/step-21/step-21.cc

```diff
@@ -1118,13 +1134,15 @@ namespace Step21
   //
   // The second point worth mentioning is that we only compute the length of
   // the present time step in the middle of solving the linear system
-  // corresponding to each time step. We can therefore output the present end
+  // corresponding to each time step. We can therefore output the present
   // time of a time step only at the end of the time step.
-  //
-  // The function as it is here does actually not compute the results
-  // found on the web page. The reason is, that even on a decent
-  // computer it runs more than a day. If you want to reproduce these
-  // results, set the final time at the end of the do loop to 250.
+  // We increment time by calling the method DiscreteTime::advance_time()
+  // inside the loop. Since we are reporting the time and dt after we
+  // increment it, we have to call the method
+  // DiscreteTime::get_previous_step_size() instead of
+  // DiscreteTime::get_next_step_size(). After many steps, when the simulation
```

Comment:
I agree that removing the dt from text output helps with the explanation here. However, because this tutorial calculates a new value of dt every step and describes this adaptive time-stepping scheme, it seems very useful to keep printing dt values.